### DISCUSSION TITLE
Unable to get past the message "Your application cannot be reviewed until you fix the following"

### DISCUSSION THREAD

**Discussion by kden** [community/community#52088] [GitHub Education] [answered]
On this page https://education.github.com/discount_requests/10082066/additional_information

I am unable to get past the message

> Your application cannot be reviewed until you fix the following:
> You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.
> You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I went and added my full name and a short bio to my regular GitHub account, so I hope that that covers the first one. 
I'm a Masters student in Wisconsin getting a degree at U Mich.  I have a student ID card.  All other documentation about my registration status is electronic.  Why can't I upload or email proof?  All I see is on the form is a way to take pictures with a webcam.   What information do you need to to receive so that I can complete this application?

Thank you

**Comment by randyb100** [comment, marked as answer]
I'm going through the same thing and the webcam function is broken. It will only let me take photos with the selfie cam. I took a photo of my portal, a letter from the National Student Clearinghouse verifying my current enrollment, and of a financial aid statement that includes my name, address, and student ID number. None of it was accepted. I'm using my student email address. I don't know how to validate that I'm a remote learner since none of the documents say that, and I don't know of any college that would say that on their documents. My university is almost solely online.

**Comment by kden** [comment]
Part of the problem I had was that it didn't seem to like me using Chrome on my desktop, so I used my phone.  As you experienced, it would not let me use the regular back camera , no matter how many times I selected the back camera from the pull down.  The ID card picture I got on the selfie cam was illegibly fuzzy.  

Finally, though, after rejecting the snapshot several times to redo it, it let me select the back camera.  I have no idea why.  At that point it let me submit the application, so at least I've gotten past that step.   So the mobile version seems to be better about accepting different locations.

**Comment by kden** [comment]
I feel like I should close this because I got past the error, but I think my experience and RandyB100's reflects that there are some issues with the web app that need to be addressed.

**Reply by W-L-W** [reply]
thanks! I wasted an hour on my laptop on that error loop but then it worked straight away on my phone. Life saver!

**Reply by kden** [reply]
The good news is that once I got past the mobile app, I got an approval email within a few days.  I still haven't been able to get copilot to work with IntelliJ but that's a story for another day.

**Comment by b2carter** [comment]
this is a joke
